Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -536,7 +536,7 @@ void UpdateBackground()
 						if (backgroundColor.IsDefault)
 							NativeView.BackgroundColor = ColorExtensions.BackgroundColor;
 						else
-							NativeView.BackgroundColor = backgroundColor.ToUIColor();
+							NativeView.BackgroundColor = Element.BackgroundColor.ToUIColor();
```

Comment:
What's the reason of this change? wasn't backgroundColor already the color of Element.BackgroundColor ?

Reply:
Just reuse the `backgroundColor` variable, but, is the same.

Reply:
right but this is another access to the can't we just leave as is ?